### ISSUE TITLE
Add a refresh button on the field mapping

### ISSUE THREAD

**Issue by MikeMillerGIS** [Esri/data-assistant#180]
If a users is hacking the xml, be nice to have a refresh to reload the file without having to select another file and picking this file again

**Comment by ChrisBuscaglia** [comment]
@MikeMillerGIS @chris-fox we shouldn't be promoting the workflow of hacking the XMLs or providing refreshes, what would you be updating that cannot be updated on this page?

**Comment by MikeMillerGIS** [comment]
services asked for this as they are repeating the same field mapping over and over for many layers.  Until we have a way to promote field mappings from config to config, they are doing it behind the scenes.  

With the UN migration, they are working with 20+ files

**Comment by previnWong** [comment]
@ChrisBuscaglia right now atleast, to disable a field from being able to be modified, you have to add the tag in the xml, no UI way to do it.

**Comment by previnWong** [comment]
ah, @MikeMillerGIS great enhancement.  when creating a new XML, have option to "use a template" and specify an existing xml file.

**Comment by MikeMillerGIS** [comment]
not really, say they map diameter, that is 40+ mappings.  That needs to be done on 24 layers in water.  They need to push that mapping to something that can be selected layer or applied to another config.
https://github.com/Esri/data-assistant/issues/172

**Comment by ChrisBuscaglia** [comment]
@MikeMiller agreed that you need to save field mappings - but what does that have to do with refresh?

**Comment by MikeMillerGIS** [comment]
lets dicuss post UC, to much going on right now, it is only in dev, it can be removed easily
